framework/Web/THttpSession.php

```diff
@@ -187,7 +187,9 @@ public function destroy()
 	public function regenerate($deleteOld = false)
 	{
 		$old = $this->getSessionID();
-		session_regenerate_id($deleteOld);
+		if (php_sapi_name() !== 'cli') {
```

Comment:
Why use http sessions at all on a CLI app? Is it because it shares the same application.xml as a web application?
This looks ok to avoid the error, but probably we could disengage the use of http sessions on CLI apps earlier.

Reply:
I'll have a look at removing httpsession from cli apps.   Great idea. I've been working on perfecting other aspects and this clearly didn't get my full attention.

Reply:
I wasn't sure how deep to look at this solution.

Reply:
Just one call seems to be what is needed.